test/Installer/testmsi.ps1

```diff
@@ -48,22 +61,26 @@ try {
     {
         throw "dotnet publish failed with exit code $LastExitCode."     
     }
-<#
-    if(-Not (Test-Administrator))
-    {
-        Write-Host -ForegroundColor Yellow "Current script testmsi.ps1 is not run as admin."
-        Write-Host -ForegroundColor Yellow "Executing MSI tests require admin privileges."
-        Write-Host -ForegroundColor Yellow "Failing silently."
-        Exit 0
-    }
-    
-    & $xunitRunner $testBin\Dotnet.Cli.Msi.Tests.exe | Out-Host
 
-    if($LastExitCode -ne 0)
+    if($env:RunInstallerTestsInDocker)
     {
-        throw "xunit runner failed with exit code $LastExitCode."       
+        CopyInstaller $testBin
+        CopyTestXUnitRunner $testBin
+
+        Write-Host "Running installer tests in Windows Container"
+        
+        $MsiFileName = [System.IO.Path]::GetFileName($inputMsi)
+        docker run `
+            -v "$testBin\:D:" `
+            -e "CLI_MSI=D:\$MsiFileName" `
+            windowsservercore `
+            D:\xunit.console.exe D:\$testName.dll | Out-Host
```

Comment:
Good catch... What happens when we want to run this outside docker? E.g. Repro on win7?